index.html

```diff
@@ -0,0 +1,14 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p>Hi, I’m @alejandro-ortiz <br>
+        I’m interested in Rocketry, Tennis, Web Development, Electronics <br>
+        I’m currently learning Git and Javascript
+        </p>
+</body>
+</html>
```

Comment:
You might also want to configure your editor to add a trailing new line. Check out [EditorConfig](https://editorconfig.org/)